docs/source/metrics.rst

```diff
@@ -63,14 +64,64 @@ It puts a greater influence on large errors than small errors, which makes this
 .. math::
     RMSE = \sqrt{\overline{(f - o)^{2}}}
 
+.. autofunction:: _rmse
+
+Mean Absolute Error (MAE)
+-------------------------
+
+``keyword: 'mae'``
+
+``perfect score: 0``
```

Comment:
I think I would remove all redundant information from `metrics.rst`. So that would be the ``perfect score`` as well as the math, since the docstring handles it. I think the keyword and the longer description with references should stay in `metrics.rst`, while the math and score ranges can stay in the docstrings.